README.cz.md

```diff
@@ -0,0 +1,101 @@
+# sway [![](https://api.travis-ci.org/SirCmpwn/sway.svg)](https://travis-ci.org/SirCmpwn/sway) [![Donate with fosspay](https://drewdevault.com/donate/static/donate-with-fosspay.png)](https://drewdevault.com/donate?project=4)
+
+Stav českého překladu můžete zkontrolovat [zde](https://github.com/SirCmpwn/sway/issues/1318)
+
+"**S**irCmpwn's **Way**land compositor" je **rozpracovaný** i3-kompatibilní
+kompozitor pro [Wayland](http://wayland.freedesktop.org/). K dispozici jsou
+**časté otázky a odpovědi** [FAQ](https://github.com/SirCmpwn/sway/wiki).
+A diskusní [IRC kanál](http://webchat.freenode.net/?channels=sway&uio=d4)
+(#sway na irc.freenode.net).
+
+[![](https://sr.ht/ICd5.png)](https://sr.ht/ICd5.png)
+
+Pokud chcete podpořit vývoj sway, můžete přispět pomocí [SirCmpwn
+Patreon stránky](https://patreon.com/sircmpwn) nebo můžete přispět
```

Comment:
"můžete přispět" is repetitive here, and you missed a comma before "nebo". "nebo odměnou" should be enough.